api/server/router/local/image.go

```diff
@@ -357,7 +357,12 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 	buildConfig.ForceRemove = httputils.BoolValue(r, "forcerm")
 	buildConfig.MemorySwap = httputils.Int64ValueOrZero(r, "memswap")
 	buildConfig.Memory = httputils.Int64ValueOrZero(r, "memory")
-	buildConfig.ShmSize = httputils.Int64ValueOrZero(r, "shmsize")
+	// default 64MB if not provided at all
```

Comment:
`Set to default if not provided` (or remove, because `httputils.Int64ValueOrDefault()` may be descriptive enough on its own)